app/jobs/lifetime_usages/recalculate_and_check_job.rb

```diff
@@ -13,7 +13,8 @@ class RecalculateAndCheckJob < ApplicationJob
     unique :until_executed, on_conflict: :log, lock_ttl: 12.hours
 
     def perform(lifetime_usage)
-      LifetimeUsages::RecalculateAndCheckService.call(lifetime_usage:)
+      LifetimeUsages::CalculateService.call(lifetime_usage:)
```

Comment:
to be completely consistent with before, this should be `call!` I think

```suggestion
      LifetimeUsages::CalculateService.call!(lifetime_usage:)
```